src/main/resources/config.yml

```
@@ -3,19 +3,33 @@
 ############################
 
 world:
-  # mode: The warp mode to use when the player is in the region (COMMAND/LOCATION/SPAWN/WARP)
+  # mode: The warp mode to use when the player is in the region (COMMAND/PLAYER_COMMAND/LOCATION/SPAWN/WARP)
   mode: SPAWN
   # yMin: The Y coordinate at the bottom of the range (Leave empty to have no minimum)
   yMin: -1000
   # yMax: The Y coordinate at the top of the range (Leave empty to have no maximum)
   yMax: 0
   # message: The message that is displayed in chat when you are teleported
   message: "&#b7faa2Teleported to &#66b04f%location%"
-
 world_warp_example:
   mode: WARP
   yMin: -1000
   yMax: 0
   message: "&#b7faa2Teleported to &#66b04f%location%"
-  warps: []
-  whitelist: false
\ No newline at end of file
+  warps: [ ]
+  whitelist: false
+player_command_example:
+  mode: PLAYER_COMMAND
+  yMin: -1000
+  yMax: 0
+  message: "<grey>Teleported to <yellow>your island</yellow>."
```

MiniMessage doesn't work on spigot servers so legacy formats in default configs are preferred :)
```suggestion
  message: "&7Teleported to &eyour island&7."
```